index.rst

```diff
@@ -48,10 +48,63 @@
 
    **This technote is not yet published.**
 
-   This proposal outlines how observing tasks will be flowed from the planning stage through execution, then linked to data analysis technotes and/or notebooks.
+   This proposal outlines how observing tasks will be flowed from the planning stage through execution,
+   then linked to data analysis technotes and/or notebooks.
+
+Motivation for this proposal
+============================
+
+    * We want central repository for all information related to a task executed at night.
+    * We want to have a way to know how and when the tasks were run and how the data was processed.
+    * Search and find tasks and their results.
+
+Steps
+=====
+
+    * Run planning
+        * Which tasks/test are we running during a observation run?
+        * Do we want to store these tasks/tests as JIRA Tickets?
+        * If using JIRA Tickets, we should use it as a place where we centralize relational information (link to
+          notebooks, LOVE scripts configuration files, test cases, confluence log pages, etc.)
+
+    * Nighttime observation
+        * Keep the Confluence Logging Page? Yes, for now it is the easiest place for nighttime staff to record task info.
+        * Should we add the Confluence Logging Page to the AuxTel Run ticket? Yes!
+        * We split the notebooks into task instead of having a single notebook for each observing night or observing
+          run. It seems that our team is happy with this new structure since it gives flexibility to run them in a
+          different order it desired. These notebooks now live in a private repository on GitHub, which is not ideal.
+          However, this new structure might give us a better idea about how do we want to organize them.
+
+    * Daytime JIRA compilation
+        * During the daytime, somebody must be resposible for extracting relevant information from nightlog and populating
```

Comment:
Ideally this would happen on-the-fly by observers. I think we need some sort of mechanism where we can flag this as "come back to it and fill in information"